src/libs/glad/README.md

```diff
@@ -1,20 +1,11 @@
 # Glad 2
 
-This OpenGL loader has been generated by the online version of Glad 2, the
+These OpenGL loaders have been generated by the online version of Glad 2, the
 multi-Language Vulkan/GL/GLES/EGL/GLX/WGL loader-generator with the following
-parameters:
-
-- OpenGL 2.1 compatibility profile
-- Enable function pointer aliasing
-- Extensions:
-  - GL_ARB_framebuffer_sRGB
-  - GL_EXT_bgra
-
-Use this permalink to re-generate the loader with different parameters (e.g.,
-to add extensions):
-
-https://gen.glad.sh/#generator=c&api=gl%3D2.1&profile=gl%3Dcompatibility%2Cgles1%3Dcommon&extensions=GL_ARB_framebuffer_sRGB%2CGL_EXT_bgra&options=ALIAS%2CALIAS
+parameters.
```

Comment:
> with the following parameters.

This is unclear - the list of parameters is not here.

Reply:
Indeed. Fixed.